planner/core/planbuilder.go

```diff
@@ -1143,84 +1138,60 @@ func (b *PlanBuilder) buildPhysicalIndexLookUpReader(ctx context.Context, dbName
 				idxReaderCols = append(idxReaderCols, col)
 				tblReaderCols = append(tblReaderCols, col)
 				schema.Append(fullExprCols.Columns[i])
-				colsMap.Insert(col.ID)
-				if mysql.HasPriKeyFlag(col.Flag) {
-					pkOffsets = append(pkOffsets, len(tblReaderCols)-1)
```

Comment:
Why remove this?

Reply:
1. We don't need to deal with genExpr, since tableReader can output virtual generated column.
2. pkOffsets is handled after ExpandVirtualColumn()